### PR TITLE
create blog post for webassembly hub + Istio; one more try

### PR DESCRIPTION
**Pull request by christian-posta**

### CODE REVIEW

**Comment by christian-posta** [comment]
@bettyjunod @frankbu

File: content/en/blog/2020/wasmhub-istio/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Extended and Improved WebAssemblyHub to Bring the Power of WebAssembly to Envoy and Istio
+subtitle: A place to build, publish, share, and deploy WebAssembly Envoy extensions
+description: Community partner tooling of Wasm for Istio by Solo.io.
+publishdate: 2020-03-13
```
Comment (frankbu) [inline content/en/blog/2020/wasmhub-istio/index.md]:
```suggestion
publishdate: 2020-03-20
```

File: content/en/blog/2020/wasmhub-istio/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+*[Originally posted on the Solo.io blog](https://www.solo.io/blog/an-extended-and-improved-webassembly-hub-to-helps-bring-the-power-of-webassembly-to-envoy-and-istio/)*
+
+As organizations adopt Envoy-based infrastructure like Istio to help solve challenges with microservices communication, they inevitably find themselves needing to customize some part of that infrastructure to fit within their organization's constraints. [WebAssembly (Wasm)](https://webassembly.org/) has emerged as a safe, secure, and dynamic environment for platform extension.
+
+In the recent [announcement of Istio 1.5](/blog/2020/wasm-announce/), lays the foundation for bringing WebAssembly to the popular Envoy proxy. [Solo.io](https://solo.io) is collaborating with Google and the Istio community to simplify the overall experience of creating, sharing, and deploying WebAssembly extensions to Envoy and Istio. It wasn't that long ago that Google and others laid the foundation for containers, and Docker built a great user experience to make it consumable. Similarly, this effort makes Wasm consumable by building the best user experience for WebAssembly on Istio.
```
Comment (frankbu) [inline content/en/blog/2020/wasmhub-istio/index.md]:
```suggestion
In the recent [announcement of Istio 1.5](/blog/2020/wasm-announce/), the Istio project lays the foundation for bringing WebAssembly to the popular Envoy proxy. [Solo.io](https://solo.io) is collaborating with Google and the Istio community to simplify the overall experience of creating, sharing, and deploying WebAssembly extensions to Envoy and Istio. It wasn't that long ago that Google and others laid the foundation for containers, and Docker built a great user experience to make it consumable. Similarly, this effort makes Wasm consumable by building the best user experience for WebAssembly on Istio.
```

File: content/en/blog/2020/wasmhub-istio/index.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Lastly, like Docker, the WebAssembly Hub stores and distributes Wasm extensions as OCI images. This makes pushing, pulling, and running extensions as easy as Docker containers. Wasm extension images are versioned and cryptographically secure, making it safe to run extensions locally the same way you would in production. This allows you to build and push as well as trust the source when they pull down and deploy images.
+
+## WebAssembly Hub with Istio
+
+The WebAssembly Hub now fully automates the process of deploying Wasm extensions to Istio, (as well as other Envoy-based frameworks like [Gloo API Gateway](https://https://docs.solo.io/gloo/latest/)) installed in Kubernetes. With this deployment feature, the WebAssembly Hub relieves the operator or end user from having to manually configure the Envoy proxy in their Istio service mesh to use their WebAssembly modules.
+
+Take a look at the following video to see just how easy it is to get started with WebAssembly and Istio:
+[* Video Part 1](https://www.youtube.com/watch?v=-XPTGXEpUp8)
```
Comment (frankbu) [inline content/en/blog/2020/wasmhub-istio/index.md]:
```suggestion
* [Video Part 1](https://www.youtube.com/watch?v=-XPTGXEpUp8)
```

File: content/en/blog/2020/wasmhub-istio/index.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+[* Part 2](https://youtu.be/vuJKRnjh1b8)
```
Comment (frankbu) [inline content/en/blog/2020/wasmhub-istio/index.md]:
```suggestion
* [Part 2](https://youtu.be/vuJKRnjh1b8)
```

**Comment by christian-posta** [comment]
Maybe someone could help me on this lint error? Seems a ruby error, can't tell how it's related to this content.

File: content/en/blog/2020/wasmhub-istio/index.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+---
+title: Extended and Improved WebAssemblyHub to Bring the Power of WebAssembly to Envoy and Istio
+subtitle: A place to build, publish, share, and deploy WebAssembly Envoy extensions
+description: Community partner tooling of Wasm for Istio by Solo.io.
+publishdate: 2020-03-25
+attribution: "Idit Levine (Solo.io)"
+keywords: [wasm,extensibility,alpha,performance,operator]
+---
+
+*[Originally posted on the Solo.io blog](https://www.solo.io/blog/an-extended-and-improved-webassembly-hub-to-helps-bring-the-power-of-webassembly-to-envoy-and-istio/)*
```
Comment (ericvn) [inline content/en/blog/2020/wasmhub-istio/index.md]: Your error is removed if you remove the _*'s*_ on this line.

Comment (ericvn) [inline content/en/blog/2020/wasmhub-istio/index.md]:
```suggestion
[*Originally posted on the Solo.io blog*](https://www.solo.io/blog/an-extended-and-improved-webassembly-hub-to-helps-bring-the-power-of-webassembly-to-envoy-and-istio/)
```

**Pull request merged**